src/test/datascience/notebook/helper.ts

```diff
@@ -600,10 +600,11 @@ async function waitForActiveNotebookEditor(notebookEditor?: NotebookEditor): Pro
 
 async function getActiveInterpreterKernelConnection() {
     const { interpreterService, kernelFinder } = await getServices();
-    const interpreter = await interpreterService.getActiveInterpreter();
-    if (!interpreter) {
-        assert.fail('Active Interpreter is undefined');
-    }
+    const interpreter = await waitForCondition(
+        () => interpreterService.getActiveInterpreter(),
+        defaultNotebookTestTimeout,
+        'Active Interpreter is undefined.2'
```

Comment:
Changed to try and wait for Python extension to get the actiev interpreter.
Hacky work around for Python extnesion issue https://github.com/microsoft/vscode-python/issues/20147
Doesn't seem to work always, but left this here.

Also added `.2` suffix to identify where where things were failing, left this here